scripts/linux_build.sh

```diff
@@ -90,6 +90,50 @@ shift $((OPTIND -1))
 # dependencies array to build out
 dependencies=()
 
+function add_arch_deps() {
+  dependencies+=(
+  'avahi'
+  'base-devel'
+  'cmake'
+  'curl'
+  'git'
+  'libayatana-appindicator'
+  'libcap'
+  'libdrm'
+  'libevdev'
+  'libmfx'
+  'libnotify'
+  'libpulse'
+  'libva'
+  'libx11'
+  'libxcb'
+  'libxfixes'
+  'libxrandr'
+  'libxtst'
+  'miniupnpc'
+  'ninja'
+  'nodejs'
+  'npm'
+  'numactl'
+  'openssl'
+  'opus'
+  'udev'
+  'wayland'
+)
+
+  if [ "$skip_libva" == 0 ]; then
+    dependencies+=(
+      "libva"  # VA-API
+    )
+  fi
+
+  if [ "$skip_cuda" == 0 ]; then
+    dependencies+=(
+      "cuda"  # VA-API
+    )
+  fi
```

Comment:
This isn't going to work. Check the `install_cuda` function. Currently it downloads cuda from nvidia and installs it using the run file. But as of now there is no way to tell this script to use the system installed cuda.

Reply:
> For CUDA, we can simply check if it's present using command -v nvcc.
> 
> As for creating a package, CPack does not support generating Arch Linux–friendly packages. We can create a PKGBUILD wrapper around it to just handle packaging, but I usually use this script to check builds, and think others do so too, so I feel its not necessary, what do you think?

That's fine with both aspects. The packaging is probably not that important since we provide a nightly one in our pacman-repo anyway.

Just one more thing to address with cuda.

We will need some logic to determine the line below. Perhaps the `install_cuda` function should return the path of the compiler.

`cmake_args+=("-DCMAKE_CUDA_COMPILER:PATH=${build_dir}/cuda/bin/nvcc")`